src/config.py

```diff
@@ -7,6 +7,14 @@
 
 HUGGING_CACHE = "/home/azureuser/cloudfiles/code//hugging_cache"
 
+FAISS_NOISE_PATH = '/home/azureuser/cloudfiles/code/blobfuse/meldingen/raw_data/permits/permits_related_to_noise_disturbance/noise_permits_faiss_db'
+METADATA_STORE_PATH = '/home/azureuser/cloudfiles/code/blobfuse/meldingen/raw_data/permits/permits_related_to_noise_disturbance/noise_permits_faiss_metadata.json'
```

Comment:
maybe define the path and file seperately here? to keep it similar to the rest of the PATH variables

Reply:
Changed METADATA_STORE_PATH to METADATA_STORE_FILE